js/controller.js

```diff
@@ -88,6 +130,24 @@
                 $scope.focus = "default";
             }
 
```

Comment:
Get rid of this commented out section. The controller is already getting pretty messy and we should avoid polluting it. Also, if the config values aren't being consumed in `config.js` they shouldn't be in there.